### ISSUE TITLE
[models] Light on generated meshes

### ISSUE THREAD

**Issue by coochewgames** [raysan5/raylib#1865]
When setting up a light source as demonstrated in the raylib examples, the light does not appear uniformly across multiple generated meshes.
The light is also still shown when underneath the model when the normals are in a positive Y direction.

A sample set of code has been attached to demonstrate this and a video.
[tile_sample.c.zip](https://github.com/raysan5/raylib/files/6794521/tile_sample.c.zip)

This is running on OS X 10.13 using Raylib 3.7.0 with the shaders taken from the example.

https://user-images.githubusercontent.com/61851819/125149009-a75b4100-e179-11eb-99e8-7d07babf92c4.mov

**Comment by GoldenThumbs** [comment]
I would like to add that I encountered this issue too. I don't think there's actually anything wrong with the normal generation, I don't think the normals are being passed to the shader correctly. I looked at the input vertex normal and it just looks like the vertex position of the model. Not sure when this broke.
Here's what the normals looks like when viewed through the shader: 
![image](https://user-images.githubusercontent.com/24837412/125179598-46e9f500-e1b5-11eb-938a-abfd2f987b25.png)
